[2022-09-03 16:20:18.449] [remoteagent] [info] Extension host agent started.
[2022-09-03 16:20:18.864] [remoteagent] [info] [127.0.0.1][60610cfc][ManagementConnection] New connection established.
[2022-09-03 16:20:18.948] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] New connection established.
[2022-09-03 16:20:18.953] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] <3446> Launched Extension Host Process.
[2022-09-03 17:06:25.937] [remoteagent] [info] [127.0.0.1][60610cfc][ManagementConnection] The client has reconnected.
[2022-09-03 17:06:26.072] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] The client has reconnected.
[2022-09-03 17:06:26.108] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 17:06:26.120] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 17:06:27.014] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 17:06:28.042] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:17:30.166] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:17:30.169] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:17:30.805] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:17:31.959] [remoteagent] [info] [127.0.0.1][60610cfc][ManagementConnection] The client has reconnected.
[2022-09-03 18:17:32.077] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] The client has reconnected.
[2022-09-03 18:18:35.261] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:36.409] [remoteagent] [info] [127.0.0.1][60610cfc][ManagementConnection] The client has reconnected.
[2022-09-03 18:34:37.024] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] The client has reconnected.
[2022-09-03 18:34:37.887] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.887] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.888] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.888] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.889] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.889] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.889] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.889] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.890] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.890] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.890] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.890] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.891] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.891] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.891] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:37.891] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:34:38.004] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:55:07.230] [remoteagent] [info] [127.0.0.1][60610cfc][ManagementConnection] The client has reconnected.
[2022-09-03 18:55:08.531] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:55:08.531] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:55:08.532] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:55:08.532] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 18:55:49.014] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] The client has reconnected.
[2022-09-03 19:36:10.524] [remoteagent] [info] [127.0.0.1][60610cfc][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-09-03 19:36:10.562] [remoteagent] [info] [127.0.0.1][b8ede9dd][ExtensionHostConnection] <3446> Extension Host Process exited with code: 0, signal: null.
[2022-09-03 19:36:10.562] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-09-03 19:36:15.725] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-09-03 19:36:15.726] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-09-03 19:36:16.037] [remoteagent] [info] [127.0.0.1][4658648e][ManagementConnection] New connection established.
[2022-09-03 19:36:16.254] [remoteagent] [info] [127.0.0.1][94a3aef9][ExtensionHostConnection] New connection established.
[2022-09-03 19:36:16.272] [remoteagent] [info] [127.0.0.1][94a3aef9][ExtensionHostConnection] <16741> Launched Extension Host Process.
[2022-09-03 19:36:18.325] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/ec2-user/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-03 19:41:15.727] [remoteagent] [info] New EH opened, aborting shutdown
